examples/vault/vault_examples/export.py

```diff
@@ -23,15 +26,19 @@ def main() -> None:
         name = f"Python export example {token_hex(8)}"
 
         # Create an asymmetric key with exportable set to true
-        create_response = vault.asymmetric_generate(
-            algorithm=AsymmetricAlgorithm.Ed25519, purpose=KeyPurpose.SIGNING, name=name, exportable=True
+        create_response = vault.generate_key(
+            key_type=ItemType.ASYMMETRIC_KEY,
+            algorithm=AsymmetricKeySigningAlgorithm.ED25519,
+            purpose=AsymmetricKeyPurpose.SIGNING,
+            name=name,
+            exportable=True,
         )
 
         assert create_response.result
         key_id = create_response.result.id
 
         # export with no encryption
-        exp_resp = vault.export(id=key_id, version=1)
+        exp_resp = vault.export(item_id=key_id, version=1)
```

Comment:
why this change? API field is called just `id`

Reply:
Most because `id` is a bit generic and it was shadowing the builtin `id()` Python function.